identity-apps-core/apps/x509-certificate-authentication-portal/pom.xml

```diff
@@ -51,7 +51,6 @@
         <dependency>
             <groupId>org.wso2.orbit.commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
-            <scope>provided</scope>
```

Comment:
why are we removing the scope provided?

Reply:
Commons-codec 1.4 was present and Commons-codec 1.14.0.wso2v1 was marked as provided and was excluded from being packed into the webapp before
With this change, since v1.4 is excluded, 1.14.0.wso2v1 is packed.